cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -13,9 +15,18 @@
 
 
 pytestmark = [pytest.mark.tier(2),
-              pytest.mark.uncollectif(lambda: version.current_version() < '5.7'),
+              pytest.mark.uncollectif(lambda provider: provider.one_of(EC2Provider) and
+                                      version.current_version() < '5.8'),
               pytest.mark.usefixtures("setup_provider_modscope")]
-pytest_generate_tests = testgen.generate([AzureProvider, OpenStackProvider], scope="module")
+pytest_generate_tests = testgen.generate([AzureProvider, OpenStackProvider, EC2Provider],
+                                         scope="module")
+
+
+def ec2_sleep():
+    # CFME currently obtains events from AWS Config thru AWS SNS
+    # EC2 Config creates config diffs apprx. every 10 minutes
+    # This workaround is needed until CFME starts using CloudWatch + CloudTrail instead
```

Comment:
I haven't created BZ about that because this was a limitation of the 3d party technology.
btw, it will be fixed soon because of switch to CloudWatch.